libs-scala/scala-utils/src/main/scala/com/daml/scalautil/nonempty/NonEmpty.scala

```diff
@@ -85,9 +85,27 @@ object NonEmptyColl extends NonEmptyCollInstances {
   }
 
   implicit final class ReshapeOps[F[_], A](private val nfa: NonEmpty[F[A]]) extends AnyVal {
+
+    /** See [[NonEmptyF]] for further explanation. */
     def toF: NonEmptyF[F, A] = NonEmpty.equiv[F, A](nfa)
   }
 
+  implicit final class UnReshapeOps[F[_], A](private val nfa: NonEmptyF[F, A]) extends AnyVal {
+
+    /** `x.toNE` is `(x: NonEmpty[F[A]])` but possibly shorter.  If code
+      * compiles without the call to `toNE`, you don't need the call.
+      */
+    def toNE: NonEmpty[F[A]] = nfa
+  }
+
+  implicit final class UnwrapOps[A](private val self: NonEmpty[A]) extends AnyVal {
+
+    /** `x.toNotNE` is `(x: A)` but possibly shorter. If code compiles without
+      * the call to `toNotNE`, you don't need the call.
+      */
+    def toNotNE: A = self
```

Comment:
A few name suggestions for this method in decreasing order of my preference:
* `forgetNE` because we forget that the collection is non-empty.
* `underlying`
* `toNotNE` isn't so great due to the negation in the name.
* `unwrap` is not my favorite because there technically is no wrapping / unwrapping happening at runtime. In Canton we already have far too many `unwrap` methods. Something like `x.unwrap.unwrap.unwrap.map(_.unwrap)` is just unreadable.

Reply:
`forgetNE` 3736544432e